src/app.jsx

```diff
@@ -31,17 +31,17 @@ class Application extends React.Component {
         super(props);
         this.state = {
             version: { version: "unknown" },
-            images: [],
-            containers: [],
-            imagesMeta: [],
-            containersMeta: [],
-            containersStats:[],
+            images: [], /* detail info of each image from InspectImage */
+            containers: [], /* detail info of each container from InspectContainer */
+            containersStats:[], /* memory usage of running containers */
```

Comment:
Are these really lists or object maps? Please add examples that show the structure.